lib/converters/circuit-to-3d.ts

```diff
@@ -128,24 +271,31 @@ export async function convertCircuitJsonTo3D(
         }
       } catch (error) {
         console.warn("Failed to render board textures:", error)
-        // If texture rendering fails, use the fallback color
         boardBox.color = pcbColor
       }
     } else {
-      // No textures requested, use solid color
       boardBox.color = pcbColor
     }
 
     boxes.push(boardBox)
   }
 
+  const getBoardThickness = (pcbBoardId?: string | null): number => {
+    if (pcbBoardId && boardThicknessById.has(pcbBoardId)) {
+      return boardThicknessById.get(pcbBoardId)!
+    }
+    return primaryBoardThickness
+  }
```

Comment:
there is no such thing as a panel where boards have different thicknesses, so you can revert and just rename `boardThickness` to `pcbThickness`